Teensy/src/gpio/gpio.cpp

```diff
@@ -84,3 +88,14 @@ static daq_FuelPressure_t gpio_getFuelPressure()
 {
     return 0U;
 }
+
+
+
+static void gpio_revTickerInterrupt(int revCount){
+
+    int currentTime = micros();
+
+    engineRev = (int)(1.0 / (float)(((float)currentTime/1000000.0) - ((float)timeOfLastRev/1000000.0)));
```

Comment:
Do you mean that you want to be return how many revs have happened in the last lastPoll_ms amount of time? Or will lastPoll_ms be a time in ms since the board booted and that its since that point in time (if that makes sense).

Reply:
I have pushed the changed for the other issues so check them over. But it should just be this issue left.

Reply:
From what I understand do you want me to count the amount of revs there have been since the last time the sensors were polled and then average those out to return the rpm?